### DISCUSSION TITLE
Saved properties on delete

### DISCUSSION THREAD

**Discussion by arturkhamaiun** [spatie/laravel-activitylog#760] [Q&A] [answered]
Hello, 

I use the library and it works pretty well. As I understand, when I create/update model, then in my activity_log table (properties column) I have JSON where `old` field shows data before and `attributes` field shows current data.

But in situation when I delete a model I don't have `old` field and `attributes` field shows deleted model data. Looks like it should be the other way around. I mean, on delete `old` field should have data and the `attributes` field shouldn't exist or it may be null. Am I right?

**Comment by Gummibeer** [comment, marked as answer]
Hey,

thanks for reporting. Your expectation sounds right to me!
I will check tests and source for this behavior.
But I think that this would be a breaking change so requires a new major version. 😖
So could be that it will take some time to fix it. 🤔
